test/ulid.spec.js

```diff
@@ -1,4 +1,5 @@
 const { expect } = require("chai");
+const { describe } = require("node:test");
```

Comment:
This can be removed.

Reply:
Removed. sorry I didn't notice when and why it's been added lol
Maybe it's from the vscode extension which adds missing import automatically for me.